cypress/locators/tile/locators.js

```diff
@@ -1,6 +1,6 @@
 // knobs locators
 export const PIXEL_WIDTH_SLIDER = 'input[name="pixelWidth"]';
 export const WIDTH_SLIDER = 'input[name="width"]';
-
-// component preview locators
-export const TILE = 'div[data-component="tile"]';
+export const TILE = '[data-component="tile"]';
+export const TILE_BODY = '[data-element="body"]';
+export const TILE_TITLE = '[data-element="title"]';
```

Comment:
do we need it here ? if we could use `getDataElementByValue('title')`

Reply:
we don't need it. Removed